os/drivers/task_manager/task_manager_drv.c

```diff
@@ -155,6 +155,15 @@ static int taskmgr_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 		}
 		break;
 	case TMIOC_RESTART:
+		tcb = sched_gettcb((int)arg);
+		if (tcb == NULL) {
+			tmdbg("Invalid pid\n");
+			return ERROR;
```

Comment:
Instead of ERROR, -EINVAL might be better because this error comes from invalid pid or arguments.

Reply:
Please take a look at other cases of returning ERROR.